src/Sarif/Autogenerated/StackFrame.cs

```diff
@@ -63,12 +63,18 @@ public SarifNodeKind SarifNodeKind
         public int Column { get; set; }
 
         /// <summary>
-        /// The name of the module that contains the code that is executing.
+        /// The name of the module that contains the code for this stack frame.
         /// </summary>
         [DataMember(Name = "module", IsRequired = true)]
         public string Module { get; set; }
 
         /// <summary>
+        /// The thread identifier of the stack frame.
```

Comment:
For `Module`, you wrote "for this stack frame"; here, you write "of the stack frame". I think the uses of "for" and "of" are fine. I'd prefer "... of this stack frame" for more consistency.